### DISCUSSION TITLE
real android device post method recieve 403 code

### DISCUSSION THREAD

**Discussion by xwyzsn** [quasarframework/quasar#9312] [CLI - Capacitor / Cordova modes] [answered]
## problem
I built an android app. It's all working fine on the emulator(both get and post requests). Once I build on a real android device, the get requests were fine but **bad with post requests**. I just received a 403 code 

Android: 8.1.0

my post method is basically like this
```
      axios({
        url: this.api_url+"/api/study",
        method: "post",
        data: formdata,
        headers: {
          "Content-Type": "application/x-www-form-urlencoded",
        },
      }).then(()=>{this.reload()}).catch(e=>console.log(e))
```

**Comment by TobyMosque** [comment, marked as answer]
I think there is nothing with your app, that is just your api blocking your app.

**Reply by xwyzsn** [reply]
Thank you for your help! Certainly, that's my server blocked it. 
To solve it, I just change the annotation **CrossOrigin()** from **CrossOrigin(origins="my-webapp.com")** to **CrossOrigin(value="*")** in spring which allows all domains. But I'm curious about why the get method could work well, and I also built a spa mode that works properly
